### DISCUSSION TITLE
Webpart to search from a List and display queried info within

### DISCUSSION THREAD

**Discussion by sciencegroup** [microsoft-search/pnp-modern-search#2172] [Q&A] [answered]
We have a Webpart called 'New Starters', it used the previous PnP Search to find and display data from an existing List (cross sites) - It searches for the recent newly started employees, their office locations, work roles and start dates from the List and display all the relevant info in a templated format. 

It had been working until the end of last year, something must have changed to the PnP Search or SharePoint search. Now the Webpart no longer reads the 'start date' from the List. No changes made from the search settings, query, nor the admin end Search Schema.

We need help to locate the cause or if anyone knows how to reproduce a similar Webpart (snippet view) using the current PnP Modern Search (v.4)?

![image](https://user-images.githubusercontent.com/48793461/175086905-3754b965-8844-4219-9ed4-52dcf9ef0421.png)
![image](https://user-images.githubusercontent.com/48793461/175088879-104c3037-ba1f-4e8c-9c51-d6336ec4fd42.png)
![image](https://user-images.githubusercontent.com/48793461/175090773-219e2538-0b4e-4edd-95dc-8481d46f1e0c.png)

**Comment by wobba** [comment, marked as answer]
You don't say anything about how the managed property `StartDate` is defined, or which scope. Is it an alias on a RefinableDate managed property? If so, can you try to use the RefinableDate name instead of the alias? Tested with alias and not myself without any issues, so not sure what is going on. You can try to configure a classic page using classic search web parts with the same query and see if that works as well.

And should not matter if you use v3 or v4 with the SharePoint API. The query should be the same.

**Reply by sciencegroup** [reply]
Many thanks for your suggestion! Now by calling the RefinableDate name instead of the alias it is working again! I will try to create a similar webpart using v4 soon. Thank you :)

**Reply by wobba** [reply]
You should log an issue with Microsoft support if the alias is not working for you as that is an API error.
